### ISSUE TITLE
Uncaught TypeError: Cannot read property 'offsetWidth' of null

### ISSUE THREAD

**Issue by bramus** [mcstudios/glightbox#19]
When importing GLightbox as an ES6 module (e.g. `import GLightbox from 'glightbox';`) it spits out this error when clicking a link that matches the selector:

<img width="457" alt="screen shot 2018-08-03 at 16 31 10" src="https://user-images.githubusercontent.com/213073/43648425-a6e48470-973a-11e8-8268-5efe819b4841.png">

_(No special config is used, only the `selector` option is passed into the `GLightbox` function)_

Problem has to do with `body` not being available inside the `GlightboxInit` class.

**Comment by bramus** [comment]
Nopes, no solution yet. I eventually moved away from glightbox to https://github.com/sachinchoolur/lightgallery.js

**Comment by gitrequests** [comment]
> Nopes, no solution yet. I eventually moved away from glightbox to https://github.com/sachinchoolur/lightgallery.js

Be careful with license for this plugin.

**Comment by Kelsch** [comment]
If you initiate the javascript script inside your body tag it'll work

**Comment by bramus** [comment]
> If you initiate the javascript script inside your body tag it'll work

I know. But I'm not including it in the body, I'm looking to import it as a module instead ("Modern JavaScript")

By the looks of it the culprit _(amongst other things)_ is that the code refers to `document.body` _(cached in a local `body` variable)_ instead of `window.document.body`. Didn't dig any deeper into it though, as I eventually moved to another lightbox-like implementation.

**Comment by d3pendent** [comment]
I get the same error in Chrome v72
I just include the js file and call it like this.
```
document.addEventListener("DOMContentLoaded", function() {
	GLightbox({
		selector: 'lightbox'
	});
});
```
redefining the `body` variable in the method open() is a quick workaround.

```
if(body === null){
        body = document.body;
}
```

**Comment by mcstudios** [comment]
hi @d3pendent thank's for taking a look at this, i'll update the code asap to remove the body variable as it's only used twice so there's no point. i'll let you know when the update is published.

**Comment by mcstudios** [comment]
This issue has been fixed in the latest version v2.0.0, i'm closing this for now, feel free to open any new issues if you find a bug.
